main.scss

```diff
@@ -20,3 +20,7 @@
 @import "src/scss/field-errors";
 @import "src/scss/field-valid";
 @import "src/scss/global-and-section-messages";
+
+// Set silent mode back on again to avoid it being output twice
+$o-forms-is-silent: true;
+$o-ft-forms-is-silent: true;
```

Comment:
Oh? How do we make sure modules aren't output more than once?

I've been using this technique in other modules (such as the grid https://github.com/Financial-Times/o-grid/blob/master/main.scss#L55), and the spec doesn't suggest a way to avoid multiple outputs.